adapter/README.md

```diff
@@ -4,6 +4,22 @@
 
 An adapter to easily run an [Axum](https://github.com/tokio-rs/axum) server in a Cloudflare worker.
 
+## Cloudflare workers perliminary native support for Axum on 0.0.21+
+
+Its hidden behind the feature flag "http".
```

Comment:
```suggestion
Axum support is enabled by the [http](https://github.com/cloudflare/workers-rs?tab=readme-ov-file#http-feature) feature in worker-rs. This is possible because both Axum and worker-rs use the same [http](https://docs.rs/http/latest/http/) crate.
```